python/tank/platform/engine.py

```diff
@@ -2163,6 +2164,8 @@ def _define_qt_base(self):
             else:
                 base["dialog_base"] = None
             base["wrapper"] = importer.binding
+            if importer.shiboken:
+                base["shiboken"] = importer.shiboken
```

Comment:
The `importer` object has the property `shiboken`, so I would add this to the `base` dict regardless of if it is None. I think it is better to keep the dictionary keys consistent, even if there is no shiboken module.
```suggestion
            base["shiboken"] = importer.shiboken
```